testing/cockroach/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Joe Holden <jwh@zorins.us>
+# Maintainer: Joe Holden <jwh@zorins.us>
+pkgname=cockroach
+pkgver=2.1.5
+pkgrel=1
+pkgdesc="The open source, cloud-native SQL database."
+url="https://cockroachlabs.com"
+arch="x86_64"
+license="APL2"
+makedepends="go bash cmake autoconf linux-headers ncurses-dev"
+install="$pkgname.pre-install"
+pkgusers="$pkgname"
+pkggroups="$pkgname"
+subpackages="$pkgname-openrc"
+options="!check"
+source="${pkgname}-v${pkgver}.tar.gz::https://binaries.cockroachdb.com/${pkgname}-v${pkgver}.src.tgz
+        cockroach.initd
+        cockroach.confd
+	01_no_githooks.patch"
+
+builddir=${srcdir}/${pkgname}-v${pkgver}
+
+build() {
+	cd ${srcdir}/${pkgname}-v${pkgver}
+	make CXXFLAGS="-D_BSD_SOURCE" CFLAGS="-D_BSD_SOURCE" TAGS="stdmalloc" BUILDTYPE="release" buildoss
+}
+
+package() {
+	cd ${srcdir}/${pkgname}-v${pkgver}
```

Comment:
should be `cd "$builddir"`

Reply:
I just mentioned this on IRC, for simple situations abuild should really figure it out on it's own if the builddir matches the source name, builddir is only there to appease abuild and honestly this PR is already about 40 commits (before squashing) to many